### PR TITLE
EPMDJ-4837: Update tooltip style. Add opacity prop for Icon

### PR DESCRIPTION
**Pull request by SMPlay**
Issue: EPMDJ-4837

### CODE REVIEW

File: src/components/icon/generic-icon-types.ts [hunk 1]
```diff
@@ -4,5 +4,6 @@ export interface IconProps {
   onClick?: () => void;
   viewBox?: string;
   rotate?: number;
+  opacity?: number;
```
Comment (RomanAchilov) [inline src/components/icon/generic-icon-types.ts:7]: rly?

File: src/components/tooltip/tooltip.module.scss [hunk 2]
```diff
@@ -6,14 +6,14 @@
 
   &__message {
     position: relative;
-    padding: 8px 16px;
+    padding: 10px 20px;
```
Comment (RomanAchilov) [inline src/components/tooltip/tooltip.module.scss:9]: I did this in my task about the tooltip

**Review by RomanAchilov — CHANGES_REQUESTED** [review]

File: src/components/tooltip/tooltip.module.scss (hunk 2 above)
Comment (RomanAchilov) [reply src/components/tooltip/tooltip.module.scss:9]: and 10px incorrect...

File: src/components/icon/generic-icon-types.ts (hunk 1 above)
Comment (SMPlay) [reply src/components/icon/generic-icon-types.ts:7]: Why not?

**Comment by pavyarov** [comment]
Closed after discussion with @RomanAchilov

**Pull request closed without merging**